crates/bws/src/render.rs

```diff
@@ -49,6 +50,31 @@ pub(crate) fn serialize_response<T: Serialize + TableSerialize<N>, const N: usiz
             let text = serde_yaml::to_string(&data).unwrap();
             pretty_print("yaml", &text, color);
         }
+        Output::Env => {
+            let valid_key_regex = regex::Regex::new("^[a-zA-Z_][a-zA-Z0-9_]*$").unwrap();
+
+            let mut commented_out = false;
+            let mut text: Vec<String> = data
+                .get_values()
+                .into_iter()
+                .map(|row| {
+                    if valid_key_regex.is_match(&row[1]) {
+                        format!("{}=\"{}\"", row[1], row[2])
+                    } else {
+                        commented_out = true;
+                        format!("# {}=\"{}\"", row[1], row[2].replace("\n", "\n# "))
```

Comment:
```suggestion
                        format!("# {}=\"{}\"", row[1], row[2].replace('\n', "\n# "))
```

Last change now, just to make CI happy 😄